<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(255, 248, 248)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_view2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="body_item">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>600</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">border-radius:5px;
border:5px solid #057A3A;
background-color: #057A3A;
border-bottom: 2px solid rgb(108, 108, 108);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>36</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { color:rgb(254, 253, 247);
	border-radius:15px;		
	border: 4px solid rgb(255, 255, 255);
	background-color: rgb(0, 76, 0)

 }

QPushButton:hover{background-color:#057A3A;color:white}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
 
	background-color: rgb(0, 85, 0);
 
}</string>
              </property>
              <property name="text">
               <string>bag_btn</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>36</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { color:rgb(254, 253, 247);
	border-radius:15px;		
	border: 4px solid rgb(255, 255, 255);
	background-color: rgb(0, 76, 0)

 }

QPushButton:hover{background-color:#057A3A;color:white}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
 
	background-color: rgb(0, 85, 0);
 
}</string>
              </property>
              <property name="text">
               <string>patrim_btn</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignBottom">
             <widget class="QFrame" name="frame_4">
              <property name="styleSheet">
               <string notr="true">border-bottom: 0</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton { color:rgb(254, 253, 247);
	border-radius:15px;		
	border: 4px solid rgb(255, 255, 255);
	background-color: rgb(0, 76, 0)

 }

QPushButton:hover{background-color:#057A3A;color:white}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
 
	background-color: rgb(0, 85, 0);
 
}</string>
                 </property>
                 <property name="text">
                  <string>rlt_item_btn</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cat_item_btn">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton { color:rgb(254, 253, 247);
	border-radius:15px;		
	border: 4px solid rgb(255, 255, 255);
	background-color: rgb(0, 76, 0)

 }

QPushButton:hover{background-color:#057A3A;color:white}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
 
	background-color: rgb(0, 85, 0);
 
}</string>
                 </property>
                 <property name="text">
                  <string>cat_item_btn</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QListView" name="listView">
                 <property name="styleSheet">
                  <string notr="true">background-color:white;	
border:3px solid #057A3A;
border-radius:10px

</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">margin:6px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QListView" name="listView_2">
                 <property name="styleSheet">
                  <string notr="true">background-color:white;	
border:3px solid #057A3A;
border-radius:10px

</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_3">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:white;	
border:3px solid #057A3A;
border-radius:10px

</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
